pkgs/development/python-modules/brother-ql/default.nix

```
@@ -10,36 +10,30 @@
   mock,
   click,
   attrs,
+  jsons,
   lib,
 }:
 
 buildPythonPackage rec {
   pname = "brother-ql";
-  version = "0.9.4";
-  format = "setuptools";
+  version = "0.11.2";
+  format = "pyproject";
```

```suggestion
  pyproject = true;
```

https://nixos.org/manual/nixpkgs/unstable/#buildpythonpackage-parameters

this will use newer pypa infra to build packages

---

Good catch, thanks!